CodingChallenges/CC_153_Interactive_SketchRRN/P5/rdp.js

```diff
@@ -0,0 +1,48 @@
+let epsilon = 5;
```

Comment:
```suggestion
// The Coding Train
// https://thecodingtrain.com/CodingChallenges/153-interactive-sketchrnn.html
// https://youtu.be/ZCXkvwLxBrA
// https://editor.p5js.org/codingtrain/sketches/hcumr-aua

let epsilon = 5;
```

Should the headers say "The Coding Train" instead of or in addition to "Daniel Shiffman"?